deploy/storage/bigquery.version

```diff
@@ -1 +1 @@
-datcom-store.dc_kg_2022_04_25_23_45_30
\ No newline at end of file
+datcom-store.dc_kg_2022_05_23_23_14_39
```

Comment:
Could we revert the BQ version, since we've not yet gotten a full build since the bug from last week?

![Screen Shot 2022-05-31 at 12 30 16 PM](https://user-images.githubusercontent.com/4375037/171269692-520a6878-6fac-4b0d-b7ca-aba6e8ace32a.png)

Reply:
done